src/image_modifications.cpp

```diff
@@ -38,10 +38,7 @@ namespace image {
 /** Adds @a mod to the queue (unless mod is nullptr). */
 void modification_queue::push(std::unique_ptr<modification> mod)
 {
-	// Null pointers do not get stored. (Shouldn't happen, but just in case.)
-	if(mod != nullptr) {
-		priorities_[mod->priority()].push_back(std::move(mod));
-	}
+	priorities_[mod->priority()].push_back(std::move(mod));
```

Comment:
I don't really mind this change, but it doesn't hurt to leave an error message here either.

If you're making this change, you _do_ need to make doubly sure that all the call sites checked for null before pushing. Have you done so?